### ISSUE TITLE
Link-name: Links must have discernible text

### ISSUE THREAD

**Issue by maxneuvians** [cds-snc/notification-api#99]
```
<a href="/accounts-or-dashboard" title="Go to the Canada.ca homepage" id="logo" class="content"> <img src="/static/images/fip/goc--header-logo.svg" width="250" alt=""> </a>
```

Fix the following:
- Element is in tab order and does not have accessible text
Fix ONE of the following:
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"
